packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -1671,6 +1671,20 @@ class IrisGridUtils {
 
     return { groups: [...groupMap.values()], maxDepth, groupMap, parentMap };
   }
+
+  static convertValueToText(value: unknown, columnName?: string): unknown {
+    if (
+      columnName != null &&
+      TableUtils.isCharType(columnName) &&
+      value != null
+    ) {
+      return String.fromCharCode(parseInt(value as string, 10));
```

Comment:
This isn't right - you only want to do this conversion if a number is provided, and in that case doing `value as string` doesn't make sense. Should be something like:
```suggestion
    if (
      columnName != null &&
      TableUtils.isCharType(columnName) &&
      typeof value === 'number'
    ) {
      return String.fromCharCode(value);
```

Reply:
value is being passed in as a string in this case, so we'll have to handle that explicitly then by checking converting value to number does not return NaN